terraform/network.tf

```diff
@@ -0,0 +1,17 @@
+# VPC module for setting up vpc
+module "vpc" {
+  source                     = "github.com/azavea/terraform-aws-vpc?ref=3.1.1"
+  name                       = "gtsite${var.environment}"
```

Comment:
I think that we should avoid using `gtsite` because the cluster that'll end up getting housed by this VPC will contain more than just the GeoTrellis site. I think that it should be something generic like `vpcGeoTrellisProduction`.